### PR TITLE
fix: update Kafka prices

### PR DESCRIPTION
**Pull request by riccardo-forina**
**What this PR does / why we need it**:

Updates the pricing for the Kafka Overview page V2.

**Verification steps** 

Stories for the App Services section have been reorganized a bit to reduce the number of clicks to reach them. For the above change, the relative story is App Services > Kafka Page V2
The relevant copy changes are here https://github.com/redhat-developer/app-services-ui-components/pull/339/files#diff-f1147ddb89bb5329e04408f72041aa062f1c7d6ee95183393889095bac79c49b

### CODE REVIEW

File: locales/en/kafkaoverview-v2.json [hunk 1]
```diff
@@ -5,15 +5,13 @@
   "contactSalesCardMainText": "Contact sales to get a pre-paid subscription or modify limits of your current subscription.",
   "contactSalesCardTitle": "Contact sales",
   "dataTransfer": "Data transfer",
+  "dataTransfer_price": "$0.09 / GB",
```
Comment (riccardo-forina) [inline locales/en/kafkaoverview-v2.json:8]: @jgiardino the copy in the doc was using spaces around `/`. I kept the spaces. Is this correct?

File: locales/en/kafkaoverview-v2.json [hunk 2]
```diff
@@ -5,15 +5,13 @@
...
   "heroTagline": "Fully hosted and managed service for stream-based applications",
   "heroTitle": "Get started with Red Hat OpenShift Streams for Apache Kafka",
-  "heroTryItButton": "",
   "kafkaInstanceCapacityDescription": "OpenShift Streams offers instances in the following sizes with the following capacity.",
-  "kafkaInstanceCapacityfooter1": "*Storage can be extended. If you wish to increase the storage limit for a Kafka instance, contact",
-  "kafkaInstanceCapacityfooter2": "to discuss your options. You can check your current storage usage by",
+  "kafkaInstanceCapacityfooter": "*Storage can be extended. If you wish to increase the storage limit for a Kafka instance, contact <0>Red Hat support</0> to discuss your options. You can check your current storage usage by <1>monitoring disk space metrics</1>.",
```
Comment (riccardo-forina) [inline locales/en/kafkaoverview-v2.json:14]: @HemaHG try not to split translation strings into substrings that are then concatenated back in the HTML. It makes reasoning about the copy more complicated than it should.

File: locales/en/kafkaoverview-v2.json [hunk 3]
```diff
@@ -5,15 +5,13 @@
...
   "dataTransfer": "Data transfer",
+  "dataTransfer_price": "$0.09 / GB",
   "firstStreamingUnit": "1 streaming unit",
-  "heroCallToActionButton": "",
   "heroDescription": "Red Hat OpenShift Streams for Apache Kafka is a managed cloud service that provides a streamlined developer experience for building, deploying, and scaling real-time applications in hybrid-cloud environments. The combination of seamless operations across distributed microservices, large data transfer volumes, and managed operations allows teams to focus on core competencies, accelerate application velocity and reduce operational cost.",
   "heroTagline": "Fully hosted and managed service for stream-based applications",
   "heroTitle": "Get started with Red Hat OpenShift Streams for Apache Kafka",
-  "heroTryItButton": "",
   "kafkaInstanceCapacityDescription": "OpenShift Streams offers instances in the following sizes with the following capacity.",
-  "kafkaInstanceCapacityfooter1": "*Storage can be extended. If you wish to increase the storage limit for a Kafka instance, contact",
```
Comment (riccardo-forina) [inline locales/en/kafkaoverview-v2.json:15]: @HemaHG try not to split translation strings into substrings that are then concatenated back in the HTML. It makes reasoning about the copy more complicated than it should.

File: locales/en/kafkaoverview-v2.json (hunk 2 above)
Comment (riccardo-forina) [inline locales/en/kafkaoverview-v2.json:14]: @HemaHG here the copy is complete with all the elements. We then inject the buttons with the right styling and attributes in the code.

**Review by HemaHG — APPROVED** [review]
LGTM, Thanks for making the updates @riccardo-forina

File: locales/en/kafkaoverview-v2.json (hunk 1 above)
Comment (jgiardino) [reply locales/en/kafkaoverview-v2.json:8]: I didn't see any objections to the spaces in the content document. And there were a lot of [comments](https://docs.google.com/document/d/1Csg8gNZnog_PvNlSWaZIzww8KZ0TUYhlOVCuwf7Y3gc/edit?pli=1&disco=AAAAZPM15D0) from the BU about these, where the format of `x / y` was used in their comments. From that, I'd say the spaces are expected.

File: locales/en/kafkaoverview-v2.json [hunk 4]
```diff
@@ -5,15 +5,13 @@
...
   "contactSalesCardTitle": "Contact sales",
   "dataTransfer": "Data transfer",
+  "dataTransfer_price": "$0.09 / GB",
   "firstStreamingUnit": "1 streaming unit",
-  "heroCallToActionButton": "",
   "heroDescription": "Red Hat OpenShift Streams for Apache Kafka is a managed cloud service that provides a streamlined developer experience for building, deploying, and scaling real-time applications in hybrid-cloud environments. The combination of seamless operations across distributed microservices, large data transfer volumes, and managed operations allows teams to focus on core competencies, accelerate application velocity and reduce operational cost.",
   "heroTagline": "Fully hosted and managed service for stream-based applications",
   "heroTitle": "Get started with Red Hat OpenShift Streams for Apache Kafka",
-  "heroTryItButton": "",
   "kafkaInstanceCapacityDescription": "OpenShift Streams offers instances in the following sizes with the following capacity.",
```
Comment (jgiardino) [inline locales/en/kafkaoverview-v2.json:13]: I apologize for not catching this earlier. A change was made in the navigation to use "Streams for Apache Kafka" as the short name. Can we update that here to match how it displays in the navigation?
> Streams for Apache Kafka offers instances in the following sizes with the following capacity.

**Review by jgiardino — APPROVED** [review]
I have one request for a text string change, but otherwise your updates look good.

**Pull request merged**